static/js/webrtc.js

```diff
@@ -538,9 +538,9 @@ var rtc = (function() {
             }
           });
         })
-        .catch(function() {
+        .catch(function(err) {
           var reason = "Sorry, we couldnt't find a suitable camera on your device. If you have a camera, make sure it set up correctly and refresh this website to retry.";
-          if(location.protocol !== "https:") reason = "Sorry, you need to install SSL certificates for your Etherpad instance to use WebRTC";
+          if(err.name !== "NotFoundError") reason = "Sorry, you need to install SSL certificates for your Etherpad instance to use WebRTC";
```

Comment:
I just realized this has more effectse. Beforehand, any error that was neither `NotAllowedError` nor `NotFoundError` would default to the "not found" error message. Now it'll default to the "not allowed" error message.

But, both of those are wrong. Should I just add a generic "unknown error" message?